File: asistencia/src/main/java/com/microservice/asistencia/model/Estudiante.java
package com.microservice.asistencia.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Estudiante {
    private Long id;

    private String nombre;
    private String apellidoPaterno;
    private String apellidoMaterno;
    private String codigoEstudiante;
    private Long aulaId;
    private Aula aula;
}

File: asistencias-main/asistencias-app/src/app/models/estudiante.ts
import { Aula } from "./aula";

export class Estudiante {
    id!: number;
    nombre!: string;
    apellidoPaterno!: string;
    apellidoMaterno!: string;
    dni!: string;
    codigoEstudiante!: string;
    aula!: Aula;
}
